### PR TITLE
Fix tmp ns creation

### PR DESCRIPTION
**Pull request by pameladelgado**

### CODE REVIEW

**Review by ableuler — APPROVED** [review]

**Comment by ableuler** [comment]
Also renamed the secret stored in github.

**Comment by pameladelgado** [comment]
:facepalm: 
thanks for spotting this!
I'll merge the PR as soon as the checks pass

File: .github/workflows/pull-request-test.yml [hunk 1]
```diff
@@ -87,7 +88,7 @@ jobs:
         RENKUBOT_KUBECONFIG: ${{ secrets.RENKUBOT_KUBECONFIG }}
         CI_RENKU_VALUES: ${{ secrets.CI_RENKU_VALUES }}
         RENKUBOT_RANCHER_APISECRET: ${{ secrets.RENKUBOT_RANCHER_APISECRET }}
-        CI_PROJET_ID: ${{ secrets.CI_PROJET_ID }}
```
Comment (rokroskar) [inline .github/workflows/pull-request-test.yml:90]: oh I just thought it was in french 😄

**Comment by ableuler** [comment]
> I'll merge the PR as soon as the checks pass

I took the liberty to go ahead and merge it to avoid any other PR from getting merged first :)

**Pull request merged**